packages/core/components/GlobalStyle/GlobalStyle.tsx

```diff
@@ -1,5 +1,6 @@
-import { createGlobalStyle } from '@xstyled/styled-components';
```

Comment:
what do you think to use the createGlobalStyle from xstyled?

Reply:
Changed!